cmd/mock-onelogin/main.go

```diff
@@ -173,6 +180,8 @@ func authorize() http.HandlerFunc {
 			sub:      r.FormValue("sub"),
 		}
 
+		emailOverride = r.FormValue("email")
```

Comment:
this could live in `sessionData` to avoid having the global state

Reply:
I thought this would be trickier to get back out that it turned out being - good call.